processor/src/main/java/org/mapstruct/ap/internal/util/MapperConfiguration.java

```diff
@@ -155,6 +156,15 @@ public NullValueCheckStrategyPrism getNullValueCheckStrategy() {
         }
     }
 
+    public InjectionStrategyPrism getInjectionStrategy() {
```

Comment:
I think that we need to have a test where we would test this as well. Not a unit test, but via the `@MapperConfig`